backend/alert/templates/alert/email_alert.html

```diff
@@ -6,7 +6,7 @@
         <img src="https://penncoursealert.com/assets/pca/PCA_logo.png" alt="Penn Course Alert Logo" style="width: 40px" />
     </a>
     <div style="text-align: center;">
-        <h1 style="margin-bottom: 1.5em;">{{ course }} just opened up!</h1>
+        <h1 style="margin-bottom: 1.5em;">{{ course }} just <span style="color:green;">opened up</span>!</h1>
```

Comment:
We may want to ask designers about what colors to use -- the default `red` and `green` in HTML can be pretty jarring.